templates/global_docker_compose.yml.mustache

```diff
@@ -0,0 +1,54 @@
+version: '3.5'
+
+services:
+
+{{#services}}
+  {{name}}:
+    container_name: {{container_name}}
+    image: {{image}}
+  {{#ports.0}}
+    ports:
+    {{#ports}}
+      - "{{.}}"
+    {{/ports}}
+  {{/ports.0}}
+  {{#depends_on}}
+    depends_on:
+      - {{.}}
+  {{/depends_on}}
+  {{#restart}}
+    restart: {{.}}
+  {{/restart}}
+  {{#command}}
+    command: {{.}}
+  {{/command}}
+  {{#labels.0}}
+    labels:
+    {{#labels}}
+      - "{{.}}"
+    {{/labels}}
+  {{/labels.0}}
+  {{#volumes.0}}
+    volumes:
+    {{#volumes}}
+      - "{{.}}"
+    {{/volumes}}
+  {{/volumes.0}}
+  {{#environment.0}}
+    environment:
+    {{#environment}}
+      - {{.}}
+    {{/environment}}
+  {{/environment.0}}
+  {{#networks.0}}
+    networks:
+    {{#networks}}
+      - {{.}}
+    {{/networks}}
+  {{/networks.0}}
+{{/services}}
+
+networks:
```

Comment:
The network needs to be updated according to the changes in this PR https://github.com/EasyEngine/site-command/pull/136/files#diff-a1a8b6733f1ec03172aeea1e84336b8eR54